client/ratings_reviews/RatingsReviews.jsx

```diff
@@ -99,7 +99,7 @@ class RatingsReviews extends React.Component {
   }
 
   render() {
-    if (!$.isEmptyObject(this.state.meta) && this.state.reviews.length !== 0) {
+    if (!$.isEmptyObject(this.state.meta) && this.props.info !== null) {
```

Comment:
This is the conditional stuff you can remove because of our new App-level rendering logic, right?

Reply:
Hi @matthewwrobel, yes! I was referring to this code when you mentioned the change at the top-level earlier. I will remove them in later PRs when I refactor. Thanks again!